lib/spack/spack/tag.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+"""Classes and functions to manage package tags"""
+import collections
+import copy
+import sys
+
+if sys.version_info >= (3, 5):
+    from collections.abc import Mapping  # novm
+else:
+    from collections import Mapping
+
+import spack.error
+import spack.util.spack_json as sjson
+
+
+def get_installed_package_names():
```

Comment:
Can we rename it to:
```suggestion
def _get_installed_package_names():
```
or inline it in the only place where it's used?